### PR TITLE
Add web accessible resources demo

### PR DESCRIPTION
**Pull request by dotproto**
Add an example for the current Web Accessible Resources capabilities supported in Chrome 89.

### CODE REVIEW

**Comment by dotproto** [comment]
Do not merge until 89 reaches stable.

**Comment by dotproto** [comment]
Tagging @solomonkinard for review.

File: README.md [hunk 1]
```diff
@@ -55,5 +55,21 @@ Read more on [Getting Started](https://developer.chrome.com/extensions/getstarte
         </ul>
       </td>
     </tr>
+    <tr>
+      <td style="vertical-align:top;">
+        Web Accessible Resources <br>
+        <a href="api/web-accessible-resources"><code>api/web-accessible-resources</code></a>
+      </td>
+      <td  style="vertical-align:top;">
```
Comment (solomonkinard) [inline README.md:63]: `      <td style="vertical-align:top;">`

File: api/web-accessible-resources/readme.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```
Comment (solomonkinard) [inline api/web-accessible-resources/readme.css:1]: I'm assuming this attribution is intended.

**Review by solomonkinard — APPROVED** [review]
LGTM.

File: api/web-accessible-resources/readme.css (hunk 2 above)
Comment (dotproto) [reply api/web-accessible-resources/readme.css:1]: Yep!

**Pull request merged**